### PR TITLE
Added more Vnet-to-Vnet PTF test cases

### PR DESCRIPTION
**Pull request by aputriax**
- Added Vnet-to-Vnet test plan
- Added more Vnet-to-Vnet PTF test cases
- Updated sai_dash_utils

Signed-off-by: Yuriy Harhas <yuriyx.harhas@intel.com>

### CODE REVIEW

File: test/docs/testplans/vnet.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+3. ACL operations (rules adding/deleting) per group for a stage must be handled atomically.
+4. Support ability to get all ACL rules/groups based on guid.
+5. During VNET or ENI delete, implementation must support ability to delete all mappings or routes in a single API call.
+6. Add and Delete APIs are idempotent.
+7. During a delete operation, if there is a dependency, implementation shall return error and shall not perform any force-deletions or delete dependencies implicitly.
+8. During a bulk operation, if any part/subset of API fails, implementation shall return error for the entire API.
+9. Implementation must have flexible memory allocation for ENI and not reserve max scale during initial create (e.g 100k routes). This is to allow oversubscription.
+10. Implementation must not have silent failures for APIs.
+
+More details may be found in [DASH SONiC HLD](https://github.com/Azure/DASH/blob/main/documentation/general/design/dash-sonic-hld.md#15-design-considerations).
```
Comment (chrispsommers) [inline test/docs/testplans/vnet.md:63]: Link is broken, `design` was removed from path quite some time ago.
Also, change `Azure` to `sonic-net`

File: test/docs/testplans/vnet.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+| 13-14 | Use same prefixes in CA and PA networks. | `Vnet2VnetOutboundSameCaPaIpPrefixesSinglePortTest.`<br/> `vnet2VnetOutboundRouteVnetTest`<br><br>`Vnet2VnetOutboundSameCaPaIpPrefixesTwoPortsTest.`<br/> `vnet2VnetOutboundRouteVnetTest`                                                                                                                                               | Creates single ENI with the same Customer and Physical IP address outbound configuration.<br/> Verifies configuration with bidirectional VXLAN TCP traffic.<br/>1. With underlay route<br/>2. Without underlay route |
+
+
+Original table [link](https://github.com/Azure/DASH/blob/main/documentation/general/design/sdn-features-packet-transforms.md#routing-routes-and-route-action).
```
Comment (chrispsommers) [inline test/docs/testplans/vnet.md:98]: Link is broken, `design` was removed from path quite some time ago.
Also, change `Azure` to `sonic-net`

**Review by chrispsommers — CHANGES_REQUESTED** [review]
Please fix links, thanks.

**Comment by chrispsommers** [comment]
@aputriax Can we resolve the final comments? Seems like a trivial edit can finish this up. Thanks.

**Comment by yharhasx** [comment]
@chrispsommers links were fixed. Also more test cases were added with some minor fixes. If everything seems good for you, please merge.
Thanks!

**Review by chrispsommers — APPROVED** [review]

**Review by chrispsommers — APPROVED** [review]
LGTM, will be nice to get some evidence the skipped test cases work on a real target since bmv2 is lacking underlay.

**Pull request merged**